libs/windy-sounding/types/client/idbInstances.d.ts

```
@@ -0,0 +1,71 @@
+import type { AlertResponse, AlertRequest } from '@windy/alerts';
+import type { MarkedNotams } from '@plugins/airport/airport';
+import type { MinifestObject } from '@windy/Calendar.d';
+import type { SeenArticle } from '@windy/startup.d';
+import type { PromoInfoObject } from '@windy/promo';
+import type { AnyColorIdent, UserColor } from '@windy/Color.d';
+export declare const initPromise: Promise<IDBDatabase>;
+/** Db of stored user favourites */
+export declare const userFavsIdb: IDB<
+  string,
+  Fav,
+  import('../../types/interfaces').LatLon & {
```

ESLint

[error] `import()` type annotations are forbidden.

(@typescript-eslint/consistent-type-imports)